cmd/helper.go

```diff
@@ -73,6 +73,7 @@ var helperCmd = &cobra.Command{
 
 		port, _ := cmd.Flags().GetUint32(portFlag)
 		startChroot, _ := cmd.Flags().GetBool("start-chroot")
+        startChroot = startChroot || setupHost
```

Comment:
is startChroot used at all anymore? The func can probably just go

Reply:
It's used only in a helper function to do a single call with chroot. It's never passed as a flag anywhere in k8s workflow.

I will make tickets today to clean up k8s helper part of the project.